pydoctor/templates/common.html

```diff
@@ -5,51 +5,75 @@
   <head>
     <title><t:slot name="title">Something</t:slot> : API documentation</title>
     <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
-    <link rel="stylesheet" type="text/css" href="apidocs.css" />
-    <script t:render="ifusesorttable" type="text/javascript"
-            src="sorttable.js"></script>
-    <script t:render="pydoctorjs" type="text/javascript"
-            src="pydoctor.js"></script>
```

Comment:
I guess you can remove the .js files (and supporting code) unless there are other references to them.